test/sagemaker_tests/tensorflow/inference/test/integration/local/conftest.py

```diff
@@ -20,7 +20,7 @@
 def pytest_addoption(parser):
     parser.addoption('--docker-base-name', default=TFS_DOCKER_BASE_NAME)
     parser.addoption('--framework-version', default=FRAMEWORK_LATEST_VERSION, required=True)
-    parser.addoption('--processor', default='cpu', choices=['cpu', 'gpu'])
+    parser.addoption('--processor', default='cpu', choices=['cpu', 'gpu', 'eia'])
```

Comment:
Remove this choice for "eia". Argparser should throw an Exception if the choice is "eia", because there are no SageMaker Local Tests for it.